content/en/deployment/overview.md

```diff
@@ -8,3 +8,130 @@ categories: ["Canary Analysis", "Features", "Concepts"]
+Within a deployment, you can define deployment targets that are equivalent to the environments you are
+deploying to.
+
+For example, you may define staging, production-west, and production-east deployment targets.
+
+You can configure each deployment target/environment to use a [unique strategy]({{< ref "deployment/strategies/overview" >}})
+to orchestrate how your code is deployed and your traffic is routed.
+
+#### Deployment Target Constraints
+You can also configure your deployment targets to use constraints that prevent a deployment from beginning or completing until certain 
+conditions are met. For example, you can configure your deployment to wait for your code to be deployed to your staging
+environment before promoting that code to production.
+
+Constraints can be set as either `beforeDeployment` or `afterDeployment` depending on your use-case. 
+
+Information on configuring constraints can be found in the [CD-as-a-Service reference docs]({{< ref "reference/deployment/config-file/targets#targetstargetnameconstraints" >}})
+
+CD-as-a-Service offers you multiple constraint options including: 
+##### DependsOn
+The dependsOn constraint allows you to specify environments that must successfully complete prior to starting this target's deployment. 
+
+
+##### Manual Approvals and Pauses
+
+The pause constraint allows you to pause a deployment for a set period of time or until an authorized user issues an approval. 
+
+You can configure a  Kubernetes deployment target like this: 
+
+```yaml
+targets:
+  target-1: #a user-defined name to reference this deployment target
+    account: <your-remote-network-agent-identifier> # the name you gave the RNA when you installed it in your prod cluster
+    namespace: <your-namespace-prod> # defined in namespace-prod.yaml
+    strategy: <the-name-of-the-strategy> # strategy will be defined below in the strategies block      
+  target-2: #a user-defined name to reference this deployment target
+    account: <your-remote-network-agent-identifier> # the name you gave the RNA when you installed it in your prod cluster
+    namespace: <your-namespace-prod> # defined in namespace-prod.yaml
+    strategy: <the-name-of-the-strategy> # strategy will be defined below in the strategies block      
+    constraints:
+      dependsOn: ["target-1"] #A constraint indicating that the deployment to target-1 must complete successfully prior to beginning the deployment to target-2
+      beforeDeployment: # a list of constraints that must be satisfied before the deployment to the target can begin 
+        - pause: # a pause constraint that will pause the execution of the deployment until an authorized user gives an approval
+            untilApproved: true
+```
+### Deployment Manifests
+
+Deploying your code with Armory CD-as-a-Service requires telling the CD-as-a-Service platform what artifacts should be 
+deployed. To accomplish this 
+you need to provide your Kubernetes manifests to Armory. 
```

Comment:
```suggestion
To deploy your software using CD-as-a-Service, you need tell CD-as-a-Service where to find your Kubernetes manifests.
```

[Use simple and direct language](https://docs.armory.io/contribute/create/guides/docs-style-guide/#use-simple-and-direct-language). Our readers have various English language skills.